src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -159,7 +174,7 @@ public function __construct(ParameterBagInterface $parameterBag = null)
     }
 
     /**
-     * @var \ReflectionClass[] a list of class reflectors
+     * @var array<string, \ReflectionClass> a list of class reflectors
```

Comment:
what about removing all descriptions on the modified lines? they don't add value to me

Reply:
Done